functions/Invoke-DbaDbDataGenerator.ps1

```diff
@@ -300,7 +300,7 @@ function Invoke-DbaDbDataGenerator {
                         Write-ProgressHelper -StepNumber ($stepcounter++) -TotalSteps $tables.Tables.Count -Activity "Generating data" -Message "Inserting $($tableobject.Rows) rows in $($tableobject.Schema).$($tableobject.Name) in $($db.Name) on $instance"
 
                         if ($tableobject.TruncateTable) {
-                            $query += "TRUNCATE TABLE [$($tableobject.Schema)].[$($tableobject.Name)];`n"
+                            $query = "TRUNCATE TABLE [$($tableobject.Schema)].[$($tableobject.Name)]"
```

Comment:
I agree with remplacing the `+=` with just `=` but it's tidier code if the query end with a `;` to make sure it's compliant for future code when that is mandatory.

Reply:
Ok, agreed. Will change that.